index.js

```diff
@@ -44,6 +44,52 @@ function fastifyView (fastify, opts, next) {
     return page
   }
 
+  // Gets template as string from LRU cache or filesystem.
+  const getTemplateString = function (file, callback) {
+    let data = lru.get(file)
+    if (data && prod) {
+      callback(null, data)
+    } else {
+      readFile(join(templatesDir, file), 'utf-8', (err, data) => {
+        if (err) {
+          callback(err, null)
+          return
+        }
+        lru.set(file, data)
+        callback(null, data)
+      })
+    }
+  }
+
+  // Gets partials as collection of strings from LRU cache or filesystem.
+  const getPartials = function (page, partials, callback) {
+    let partialsObj = lru.get(`${page}-Partials`)
+
+    if (partialsObj && prod) {
+      callback(null, partialsObj)
```

Comment:
you should probably return after this

Reply:
Standard is barking at me when I add a return on this one: ```unnecessary return statement```. Since this callback only happens when the result is already in the cache, we won't get into the other condition and there is no risk of another call to the callback.